src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
+            if (line.startsWith("type")) {
+                continue;
+            }
+            String[] parts = line.split(",");
```

Comment:
The delimiter ',' used in the split method is a 'magic value' and should be defined as a constant to improve maintainability and avoid duplication.